src/redux/actions/componentConfig/updatePlotData.js

```diff
@@ -0,0 +1,11 @@
+import { UPDATE_CONFIG } from '../../actionTypes/componentConfig';
+
+const updatePlotData = (plotUuid, dataChange) => (dispatch) => {
```

Comment:
why couldn't the already existing `updatePlotConfig` action be reused here? Why are we creating a new action for this?

Reply:
Because `updatePlotConfig` was being used for the config side.
I could refactor `updatePlotConfig` to take `configChange` and `DataChange` instead though, and reuse it in `updatePlotData`, let me know which way would be better.